vms/proposervm/post_fork_block.go

```diff
@@ -155,12 +196,14 @@ func (b *postForkBlock) verifyPostForkOption(ctx context.Context, child *postFor
 }
 
 // Return the child (a *postForkBlock) of this block
-func (b *postForkBlock) buildChild(ctx context.Context) (Block, error) {
+func (b *postForkBlock) buildChild(ctx context.Context, blsSignKey *bls.SecretKey) (Block, error) {
```

Comment:
Isn't the bls key already accessible through the postForkBlock's reference to the VM?